pgbouncer-mixin/alerts.libsonnet

```diff
@@ -0,0 +1,92 @@
+{
+  new(this): {
+
+    groups: [
+      {
+        name: this.config.uid,
+        rules: [
+          {
+            alert: 'HighNumberClientWaitingConnections',
+            expr: |||
+              pgbouncer_pools_client_waiting_connections > %(alertsHighClientWaitingConnections)s
+            ||| % this.config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'May indicate a bottleneck in connection pooling where too many clients are waiting for available server connections.',
+              description: |||
+                The number of clients waiting connections on {{ $labels.instance }} is now above %(alertsHighClientWaitingConnections)s. The current value is {{ $value | printf "%%.2f" }}.
```

Comment:
```suggestion
                The number of clients waiting for connections on {{ $labels.instance }} is now above %(alertsHighClientWaitingConnections)s. The current value is {{ $value | printf "%%.2f" }}.
```

Reply:
Not sure what the proper grammar here is for PGBouncer but it sounds like it should be something like this or maybe 'client waiting connections' like in the threshold name.